src/time.rs

```diff
@@ -32,17 +32,15 @@ impl Instant {
         Instant { millis }
     }
 
-    /// Create a new `Instant` from the current `SystemTime`.
-    #[cfg(feature = "std")]
-    pub fn now() -> Result<Instant, ::std::time::SystemTimeError> {
-        Self::from_system_time(::std::time::SystemTime::now())
+    /// Create a new `Instant` from a number of seconds.
+    pub fn from_secs(secs: i64) -> Instant {
+        Instant { millis: secs * 1000 }
     }
 
-    /// Create a new `Instant` from a `SystemTime`.
+    /// Create a new `Instant` from the current `SystemTime`.
```

Comment:
Nit: it's not clear from the signature and doc comment that this means `std::time::SystemTime`.